src/components/Animal.js

```diff
@@ -0,0 +1,11 @@
+const List = ({ list = [] }) => {
```

Comment:
since the prop is getting values from the parent component, you don't have to assign it to an empty array